packages/shopware-6-client/src/services/productService.ts

```diff
@@ -38,6 +43,25 @@ export const getProducts = async function (
   return resp.data;
 };
 
+/**
+ * Get default amount of products for given category/listing
+ *
+ * @throws ClientApiError
+ * @alpha
+ */
+export const getListingProducts = async function (
```

Comment:
as we need to provide categoryId for that maybe we shou;d call it like `getCategoryProducts` ?

Reply:
let's meet in halfway: `getCategoryProductsListing` :D